packages/slice-machine/src/features/onboarding/OnboardingStepDialog/OnboardingStepDialog.tsx

```diff
@@ -0,0 +1,75 @@
+import {
+  Dialog,
+  DialogActions,
+  DialogContent,
+  DialogHeader,
+  ScrollArea,
+  Text,
+  Video,
+} from "@prismicio/editor-ui";
+
+import {
+  useOnboardingProgress,
+  useOnboardingStepsContent,
+} from "@/features/onboarding/helpers";
+import type { OnboardingStep } from "@/features/onboarding/types";
+
+import styles from "./OnboardingStepDialog.module.css";
+
+type OnboardingStepDialogProps = {
+  step: OnboardingStep;
+  isOpen: boolean;
+  onClose: () => void;
+};
+
+export const OnboardingStepDialog = ({
+  step,
+  isOpen,
+  onClose,
+}: OnboardingStepDialogProps) => {
+  const { toggleStepComplete, isStepComplete, getStepIndex } =
+    useOnboardingProgress();
+  const content = useOnboardingStepsContent();
+  const stepContent = content[step.id];
+
+  const markAsDone = () => {
+    if (!isOpen) return;
+    toggleStepComplete(step.id);
+    onClose();
+  };
+
+  return (
+    <Dialog open={isOpen} onOpenChange={onClose}>
+      <DialogHeader title="Learn" />
+      <DialogContent>
+        <ScrollArea>
+          <article className={styles.scrollableContent}>
```

Comment:
💡 #idea

This article is making one more wrapper, we could just use `ScrollArea` no? You can see the usage of `ScrollArea` in `editor` project with unlined style

One goal we try to have in to prevent using CSS modules in features folder, something we didn't really do before, but that can be a good indication that something is wrong.

Reply:
The reason to use the inner container here is that by styling the ScrollArea with a property like padding, the scrollbar will get pushed inwards. 

What would be the correct approach to apply styles to these components? Create them in the components folder?

Reply:
Solved: Refactored to a Box